services/app-web/src/components/ActionButton/ActionButton.module.scss

```diff
@@ -1,5 +1,5 @@
-@import '../../styles/uswdsImports.scss';
-@import '../../styles/custom.scss';
+@use '../../styles/custom.scss' as custom;
+@use '../../styles/uswdsImports.scss' as uswds;
```

Comment:
Question: is the uswds import necessary? I know it's not newly added in this PR but it doesn't seem to be used in this file